src/vs/workbench/contrib/terminal/browser/terminalEditorService.ts

```diff
@@ -163,13 +163,14 @@ export class TerminalEditorService extends Disposable implements ITerminalEditor
 		if (resource) {
 			await this._editorService.openEditor({
 				resource,
+				description: instance.shellLaunchConfig.description,
 				options:
 				{
 					pinned: true,
 					forceReload: true,
 					preserveFocus: editorOptions?.preserveFocus
 				}
-			}, editorOptions?.viewColumn);
+			}, editorOptions?.viewColumn || ACTIVE_GROUP);
```

Comment:
Is this not implied when undefined?

Reply:
I would've thought so, but apparently not